### ISSUE TITLE
Releted To Port

### ISSUE THREAD

**Issue by HirenPatelhv** [ElectronNET/Electron.NET#155]
Each time application is start using different port on every plateforn, so it's very difficult in api authentication and for CORS.

**Comment by GregorBiswanger** [comment]
Do you want access from a external desktop application?

For internal access, you need the BridgeSettings.WebPort-Property:
`string path = $"http://localhost:{BridgeSettings.WebPort}/Home/FooBar"`

**Comment by HirenPatelhv** [comment]
I Want to change AspCoreBackend port and fixed it to some static port.

**Comment by GregorBiswanger** [comment]
The default Port is 8000... is this not available, we search automatically the next open port to upper...

Do you need a other solution? I can add this as a feature request... otherwise all pull request are very welcome :)

**Comment by HirenPatelhv** [comment]
**Yes** , I want other solution ,Can you provide a way to fixed it to some **static port** for all platform (eg. Windows, Mac-OS, Linux) and that port is given by me.

I've developed an app using .NET Core and Electron.NET, also developed one API in .NET Core.
When I call API from it , Its give an error about "**Cross-Origin**", so i need to register an IP Address and  Port in that API, for that i want static port.

**Comment by HirenPatelhv** [comment]
I think you have to change

`public static string WebPort{ get; internal set; }` to  `public static string WebPort{ get; set; }`
in 
BridgeSettings class.

**Comment by HirenPatelhv** [comment]
How much time you will take for this solution if you do this.

**Comment by HirenPatelhv** [comment]
Hi @GregorBiswanger , Your Answer is very important for me.please help me to solve this issue.

**Comment by GregorBiswanger** [comment]
I think in three weeks.. sorry...

**Comment by HirenPatelhv** [comment]
thanks @GregorBiswanger , I'll wait for this feature...

**Comment by HirenPatelhv** [comment]
Hi @GregorBiswanger , What is the status of this Feature.

**Comment by HirenPatelhv** [comment]
Hi @GregorBiswanger ...

**Comment by HirenPatelhv** [comment]
Hi @GregorBiswanger , I'm still waiting for this feature.

**Comment by JimmyTheJ** [comment]
Hi @HirenPatelhv I have pushed a fix for this in my fork of this repo. I don't know if it will ever get picked up by the main repository though. If you want help getting it working I can do that.

**Comment by GregorBiswanger** [comment]
Is the feature still needed?

**Comment by HirenPatelhv** [comment]
Yes @GregorBiswanger , still I waiting for this feature

**Comment by GregorBiswanger** [comment]
Hi @HirenPatelhv,

finally I have managed to implement your feature request. You now need an additional **configuration** in the **electron.manifest.json** file with the key **aspCoreBackendPort**:

```
{
  "executable": "ElectronNET.WebApp",
  "splashscreen": {
    "imageFile": "/wwwroot/assets/img/about@2x.png"
  },
  "singleInstance": false,
  "aspCoreBackendPort": 8080, <--- Your custom Port for ASP.NET Backend
  "build": {
    "appId": "com.electronnetapidemos.app",
...

```

The application then runs automatically as a single instance. I hope you like it that way.

The feature is available with the next update **5.22.14**, which will be published as soon as possible.

**Comment by enzosagretti** [comment]
Hi! Thanks for all Gregor! It´s is possible to disable ASP por. I don´t want that user can acces from browser outside electron... thanks!
